xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -922,7 +922,8 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
   GetActions(pControlNode, "ontextchange", textChangeActions);
   GetActions(pControlNode, "onfocus", focusActions);
   GetActions(pControlNode, "onunfocus", unfocusActions);
-  focusActions.m_sendThreadMessages = unfocusActions.m_sendThreadMessages = true;
+  focusActions.SetSendThreadMessages(true);
+  unfocusActions.SetSendThreadMessages(true);
```

Comment:
As far as I can see this function will never be called with `false`. So, why not remove the bool parameter and define the semantics of the function to set the "SetSendThreadMessages" property to true.